ui/app/adapters/secret-v2-version.js

```diff
@@ -36,6 +36,13 @@ export default ApplicationAdapter.extend({
     });
   },
 
+  async getSecretDataVersion(backend, id) {
+    // used in secret-edit route when you don't have current version and you need it for pulling the correct secret-v2-version record
+    let url = this._url(backend, id);
+    let response = await this.ajax(this._url(backend, id), 'GET');
```

Comment:
make network request to https://www.vaultproject.io/api-docs/secret/kv/kv-v2#read-secret-version